src/accapi/client.py

```diff
@@ -196,7 +196,7 @@ def _request_connection(
             ("B", self._broadcastingProtocolVersion),
             ("s", displayName),
             ("s", password),
-            ("i", self._updateIntervalMs),
+            ("B", self._updateIntervalMs),
```

Comment:
"i" is actually required here, this is why the padding had to be added in the other pull request.